### ISSUE TITLE
Not updating

### ISSUE THREAD

**Issue by johnmark** [ManageIQ/planet.manageiq.org#1]
planet.manageiq.org has not updated the most recent blog post - it's been >24 hours.

@mscherer @garrett

**Comment by garrett** [comment]
@mscherer is on PTO today. I would assume he may look into this Monday.

I can't do anything about it myself, as it is a deployment issue. (If there's somehow some sort of unexpected error, then I have no way of knowing what it is or fixing it... and that'd have to wait until @mscherer can see what's happening anyway. So: Monday or after still.)

**Comment by mscherer** [comment]
So the log says:

```
   == Request: /404.html
       error  build/404.html
   undefined local variable or method `google_analytics_tag' for #<Middleman::Application:0x50592800>
/srv/builder/.gem/ruby/gems/middleman-core-3.3.8/lib/middleman-core/configuration.rb:78:in `method_missing'
```

I will work on giving access to the builder to people in some way, with some doc so people cna fix stuff by themself.

**Comment by mscherer** [comment]
And the issue is that we are missing middleman-google-analytics in the Gem file.

**Comment by garrett** [comment]
Fixed in pull request #2; git ID: d224dc1b99a6cc9f8887f605898a4d97d751daf6
